services/credential-server/src/modules/signify/signifyApi.ts

```diff
@@ -105,28 +107,49 @@ export class SignifyApi {
     registryId: string,
     schemaId: string,
     recipient: string,
-    attribute: { [key: string]: string }
+    attribute: { [key: string]: any }
   ) {
     await this.resolveOobi(`${config.oobiEndpoint}/oobi/${schemaId}`);
 
     let vcdata = {};
```

Comment:
I'm not even sure why we have `vcdata` and don't just use `attribute` (should be `attributes`?) anyway